### DISCUSSION TITLE
lysmarine-hotspot

### DISCUSSION THREAD

**Discussion by gildas-declercq** [bareboat-necessities/lysmarine_gen#348] [Q&A] [answered]
Hi,

I did not find the password.

I disabled the hotspot and connected to my wifi, It is ok until my wifi is turned off, the hotspot is turned on back automatically.
is it supposed to do that ?  how can I change that ?

thanks
Gildas

**Comment by gildas-declercq** [comment]
ok I found the password "changeme", it is shown on the hotspot config.

still interested to know to enable or not the option to auto activate hotspot when wifi configured is not reachable.
and how to configure the hotspot. actually I can connect from windows11 but cannot from my android tablet even by scanning the QR code

thanks

**Comment by mgrouch** [comment, marked as answer]
Looks like you didn’t disable hotspot. 
You seems just disconnected it. 
Does your tablet support same wifi frequency band as you set up your hotspot?

**Reply by gildas-declercq** [reply]
correct. for the wifi I did not set the country and hotspot was on 5Ghz. 
I can know jump from my phone hotspot to lysmarine-hotpost. which is good after a few test. it means I can access to the boat computer automatically if the phone is failing.
I use the phone hostpot primary, as it usually have internet access. no 4G router on board so far

thanks
